build/compile/LzmaArchive.targets

```diff
@@ -115,11 +115,7 @@
       </GenerateNuGetPackagesArchiveVersion>
       <PropertyGroup>
         <NugetPackagesArchiveName>nuGetPackagesArchive.$(NuGetPackagesArchiveVersion).lzma</NugetPackagesArchiveName>
```

Comment:
Actually I'm unable to move this to the pre-build since it depends on using the dotnet.exe that exists in the stage2 directory. This means we have to build first.